docs/architecture/grpc-for-wcf-developers/wcf-bindings.md

```diff
@@ -1,11 +1,37 @@
 ---
 title: WCF bindings and transports - gRPC for WCF Developers
-description: TO BE WRITTEN
+description: How the various WCF bindings and transports compare to gRPC
 author: markrendle
 ms.date: 09/02/2019
 ---
 
 # WCF bindings and transports
 
+WCF has lots of different built in *bindings* specifying different network protocols, wire formats and other implementation details. gRPC effectively has just one network protocol and one wire format. However, you are still likely to discover that gRPC offers the best solution in most cases. What follows is a short discussion about the most relevant WCF bindings and how they compare to their equivalent in gRPC.
+
+## NetTCP
+
+WCF's NetTCP binding allows for persistent connections, small messages and two-way messaging but only works between .NET clients and servers. gRPC allows the same functionality but based on an open standard. gRPC has almost all the features of WCF NetTCP binding, although not always in the same way. For example, in WCF, encryption is controlled through configuration and handled in the framework. In gRPC, encryption is achieved at the connection level using HTTP/2 over SSL/TLS.
```

Comment:
> SSL/TLS

Hey @blowdart how do you feel about HTTP/2 connection encryption being called "SSL/TLS"? Would you prefer just "TLS"?

Reply:
Just TLS. It's not SSL.

Reply:
@markrendle Do a find/replace on "SSL/TLS" to "TLS" across all files. Worth mentioning once somewhere that TLS is the successor to SSL.